lib/BUILD

```diff
@@ -1,7 +1,8 @@
-load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+load("@rules_cc//cc:defs.bzl", "cc_binary")
+load("@pybind11_bazel//:build_defs.bzl", "pybind_library")
 load("//lib:version.bzl", "EMBAG_VERSION")
 
-cc_library(
```

Comment:
So I assume this does something fancy with the python bindings in bazel builds, but what specifically is this needed for?

Reply:
This makes it so that all the required pybind headers are provided to gcc when building the library.